tests/testsuite/cargo_alias_config.rs

```diff
@@ -73,6 +78,75 @@ fn recursive_alias() {
         .run();
 }
 
+#[cfg(unix)]
+#[cargo_test]
+fn default_args_alias() {
+    let p = project()
+        .executable(
+            Path::new("subcommands").join("cargo-test-1"),
+            r#"
+                #!/bin/bash
```

Comment:
Instead of a shell script, would you be up for updating this test to build a small cargo project that produces a binary that does the same thing? We do that in a few other places in the test suite as well, primarily for Windows testing support.

Reply:
Found a place where this happens and refactored it into cargo-test-support/src/tools.rs